microsetta_interface/implementation.py

```diff
@@ -53,6 +53,7 @@ class Source:
 ADMIN_MODE_KEY = 'admin_mode'
 LOGIN_INFO_KEY = 'login_info'
 LANG_KEY = "language"
+claim_kit_name_hint = None
```

Comment:
This is only used in the scope of the get_source function, please move it into that scope.

Reply:
sure doing